Client/Clique/Modules/Settings/SecurityViewController.swift

```diff
@@ -0,0 +1,184 @@
+    @IBOutlet weak var repeatNewPasswordTextField: UITextField!
+    @IBOutlet private weak var passwordCheckLabel: UILabel!
+    @IBOutlet private weak var matchingPasswordsLabel: UILabel!
+    
+    var iconClick = false
+    let oldPasswordShowButton = UIButton(type: .custom)
+    let newPasswordShowButton = UIButton(type: .custom)
+    let repeatNewPasswordShowButton = UIButton(type: .custom)
+    
+    @IBOutlet weak var savePasswordButton: UIButton!
+    private let settingsService = SettingService()
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        oldPasswordTextField.isSecureTextEntry = true
+        newPasswordTextField.isSecureTextEntry = true
+        repeatNewPasswordTextField.isSecureTextEntry = true
+
+        self.showButton()
+        
+        newPasswordTextField.addTarget(self, action: #selector(checkAndDisplayError(textfield:)), for: .editingChanged)
+        repeatNewPasswordTextField.addTarget(self, action: #selector(compareAndDisplay(textfield:)), for: .editingChanged)
+    }
+    
+    @IBAction func closeSettingsViewController(_ sender: UIButton) {
+        dismiss(animated: true, completion: nil)
+    }
+    
+    func getPasswordData() -> PasswordData? {
+        
+        guard
+            let oldPassword = oldPasswordTextField.text,
+            let newPassword = newPasswordTextField.text,
+                !oldPassword.isEmpty && !newPassword.isEmpty
+        else {
+            return nil
+        }
+        
+        let newPasswordData = PasswordData(OldPassword: oldPassword, NewPassword: newPassword)
+        return newPasswordData
+    }
+        
+    func updatePasswordUser(with userPasswords: PasswordData) {
+        settingsService.changePassword(with: userPasswords) { (isSuccess) in
+            if isSuccess{
+                self.alert(fwdMessage: "Successfully updated password!")
+                self.dismiss(animated: true, completion: nil)
+            }else{
+                self.alert(fwdMessage: "Please enter all required info.")
+            }
+        }
+    }
+    
+    @IBAction func saveButtonPressed(_ sender: UIButton) {
+        
+        guard let getPsswdData = getPasswordData() else {
+            alert(fwdMessage: "Please enter all required info.")
+            return
+        }
+        
+        updatePasswordUser(with: getPsswdData)
+        
+    }
+    
+    func alert(fwdMessage: String){
+        let alertController = UIAlertController(title: "", message: fwdMessage , preferredStyle: .alert)
+        let defaultAction = UIAlertAction(title: "OK", style: .cancel, handler: nil)
+        alertController.addAction(defaultAction)
+        self.present(alertController, animated: true, completion: nil)
+    }
+    
+    func checkPasswords() -> Bool {
+        var check = false
+
+        if(newPasswordTextField.text?.count ?? 0>=8 && newPasswordTextField.text == repeatNewPasswordTextField.text){
```

Comment:
you do not need brackets for if statements, add proper spacings and new lines. You should guard newPasswordTextField.text so you do not have that many optionals.